processor/memorylimiter/internal/cgroups/cgroup.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
Hi @pavolloffay  - Is this part of the code that you expect to host somewhere else? If not we will need to clear the headers per https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/502#issuecomment-667212938

Reply:
This code is from https://github.com/uber-go/automaxprocs which is MIT license

Reply:
@open-telemetry/technical-committee any guidance about handling code with MIT license?

@pavolloffay - I will block merging until this question is cleared but will keep reviewing it so we can make progress meanwhile.